guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -34,9 +34,18 @@ There are no upgrade warnings with Foreman {ProjectVersion}.
 [id="foreman-deprecations"]
 == Deprecations
 
-ifndef::foreman-el,katello[]
+ifndef::foreman-deb,foreman-el,katello[]
```

Comment:
No idea if that's really necessary, I don't think neither Orcharino nor Satellite is using this file for their Release Notes, but it keeps the HTML diff small, so :woman_shrugging: